soh/soh/Network/Anchor/Packets/UpdateRoomState.cpp

```diff
@@ -49,6 +51,7 @@ void Anchor::HandlePacket_UpdateRoomState(nlohmann::json payload) {
 
     roomState.ownerClientId = payload["state"]["ownerClientId"].get<uint32_t>();
     roomState.pvpMode = payload["state"]["pvpMode"].get<u8>();
+    roomState.pvpDamageMult = payload["state"]["pvpDamageMult"].get<u8>();
```

Comment:
If this client joins a room that is hosted by an older client (without this setting) this will result in a crash. Could you do a conditional check here? I think there is probably some examples somewhere, if not here then in the older anchor code

Reply:
Good find, I'll probably migrate to using this in a few other places as well.